### DISCUSSION TITLE
채팅봇 질문 드립니다

### DISCUSSION THREAD

**Discussion by uNadayo** [Lastorder-DC/chatreader-kor#62] [문의] [answered]
스트리머의 채팅을 안읽게 할 수 있나요?
시청자는 채팅을 보고있을거고 내가 쓴걸 굳이 들을 필요는 없다고 생각해서
밴도 해보고 화이트리스트도 시청자만 해보았는데 계속 읽네요

**Comment by Lastorder-DC** [comment]
스트리머는 밴리스트 / 화이트리스트의 영향을 받지 않게 되어 있습니다. 관련해서 기능 추가해보겠습니다.

**Comment by Lastorder-DC** [comment, marked as answer]
기능 추가되었습니다. `!!tts vieweronly on`으로 시청자 전용 모드를 활성화하실수 있습니다.
활성화후 스트리머의 채팅만 읽지 않습니다. 이는 기존 구독 전용 / 개설자 전용 모드와 별개로 동작합니다.
